wiki/Tournaments/Official_support/en.md

```diff
@@ -187,7 +187,7 @@ We have historically had issues where unscrupulous organisers have geared tourna
 
 ### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}
 
-No. Players who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity with the sole exception of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-managerial position (referees, mappool selectors, mappool playtesters, custom mappers, judges, etc.) This list is not exhaustive. If you have any doubts about any role not noted here, please contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

Comment:
Un-needed repetition (though unsure if I'm doing this linking right, a wiki guy will have to confirm)

```suggestion
Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any of the non-managerial positions [mentioned above](#what-is-considered-staff).
```

Reply:
the current text also contradicts itself. i've bolded those parts:

> To clarify, we consider organisers, **referees, mappool selectors, mappool playtesters, custom mappers, judges, or any other form of managerial position** directly involved in the maintenance and operation of the tournament/contest to be a part of its staff.

> any **non-managerial position (referees, mappool selectors, mappool playtesters, custom mappers, judges**, etc.)

---

if i read your suggestion correctly, then

> Players who have been eliminated from a tournament may be enlisted as staff to assist in any of the non-managerial positions [mentioned above](#what-is-considered-staff)

could really only sort of be interpreted to refer to

> Commentators, streamers, and graphic designers are designated as **helpers**, and may participate in tournaments/contests freely so long as they are not directly involved in any organisational aspect of the tournament/contest.

which would mean the same thing as before this pr.

this page draws the distinction between helpers and staff but doesn't clarify what a *managerial* and *non-managerial* is. i believe that was peppy's point as well.

Reply:
This should be changed indeed.
Any mangerial role is basically anything above these roles, so mostly organizers/admins really. I don't remember any other role that would be above the other roles noted out, so we could change it from this:

> any non-managerial position (referees, mappool selectors, mappool playtesters, custom mappers, judges, etc.)

to this: 

> any non-organizational position (Organizers, admins...)

I feel like these would be the only ones incorporated, if anyone can  let me know if I did it would be nice.